draft-ietf-quic-transport.md

```diff
@@ -1208,14 +1207,16 @@ Receivers send ACK frames to inform senders which packets they have received, as
 well as which packets are considered missing.  The ACK frame contains between 1
 and 256 ACK blocks.  ACK blocks are ranges of acknowledged packets.
 
-To limit the ACK blocks to the ones that haven't yet been received by the
-sender, the sender periodically sends STOP_WAITING frames that signal the
-receiver to stop acknowledging packets below a specified sequence number,
-raising the Least Unacked packet number at the receiver.  A sender of an ACK
-frame thus reports only those ACK blocks between the received Least Unacked and
-the reported Largest Acknowledged packet numbers.  The endpoint SHOULD raise the
-Least Unacked communicated via future STOP_WAITING frames to the most recently
-received Largest Acknowledged.
+To limit ACK blocks to those that haven't yet been received by the sender,
+the receiver SHOULD track which ACK frames have been acknowledged by its peer.
+Once an ACK frame has been acknowledged, the packets it acknowledges SHOULD not
+be acknowledged again.  In order to handle cases where the receiver is not
+sending retransmittable data, and hence does not expect to receive acks, it
+MAY bundle a PING frame with an ack at most once per RTT to gain confirmation
+of ACKs being delivered.  In order to limit receiver state or the size of ACK
+frames, a receiver MAY limit the number of ack ranges and stop acknowledging
+packets without getting an ACK acked, with the knowledge this may slightly
+increase spurious retransmits.
```

Comment:
I would suggest using "ACK frames" for the noun and "acknowledge" for the verb in general. Suggestion on this particular sentence: see if you can break it up into two sentences. It's a bit of a run-on.